### DISCUSSION TITLE
longer interval Timelapse videos

### DISCUSSION THREAD

**Discussion by ujkanishka** [gopro/labs#1085] [Feature Requests]
hi,
For an art project I am undertaking, I want to take a shot from a fiXed position over a year. Each day, I want a shot of around 2-3 minutes to be taken. I was wondering if this kind of a script can be developed.

Additionally, I would appreciate any suggestions - especially because this fiXed position is from a wall of a man made lake. I intend to fix the setup at a particular point. and connect it to high capacity battery pack. There were two options I was looking at - one is the underwater Suptig one, and there's this other voltaic one which doesn't switch off automatically. I am also considering putting some dehumidifying thing. The place I want to do is in India, and it tends to get quite hot. And humid. So wondering what else precautions can be taken.

But coming back to it, it will be really greatly helpful, if one could take long interval Timelapse videos. 

thanks
ujjwal

**Comment by dnewman-gpsw** [comment]
Can explain more what you need "Each day, I want a shot of around 2-3 minutes to be taken." 2-3 minutes of what, video, video timelapse, photo time lapse, etc.  what parts of the day?

**Comment by ujkanishka** [comment]
I want to fiX the camera at one point for a year. And it should take a video something like 2-4 minutes everyday. There are two options I am considering:
A. To take it a fiXed time everyday. (so basically at an interval of 24 hours) 
B. To take it at different times of the day. (like maybe if the interval between the shots is 24 hours and 5 minutes, then it keeps shifting the time of the day on each day basis). 

I think I should be able to service the camera every now and then. (like to change the power bank and if needed transferring the data). But maybe every few months or so.

**Reply by dnewman-gpsw** [reply]
It is usual, but easy.  Use HERO12 or HERO13 for the greatest reliability 

Use a script like: ```!12:00NmV!S!150E!2R```

Sleep until noon. 
Mode Video
Start 
End after 150s
Repeat

**Reply by ujkanishka** [reply]
I was thinking of using the 12, thanks. 

Could you tell me where I could find this script, or how to eXecute this?

thanks
u.

**Reply by ujkanishka** [reply]
sorry if I am asking something too obvious or stupid.

**Reply by dnewman-gpsw** [reply]
QR codes are just text.  So any QR code generator can be used. https://gopro.github.io/labs/control/custom/ or just copy and paste into the additional commands fields using this link.

**Reply by ujkanishka** [reply]
Thanks a lot for guiding me through the process. 
I tried putting the settings on this link and the additional commands, which you had shared earlier.

When pointing the GoPro to that QR code, it says invalid QR code and then after a while it switches off.

Is this how it should react? How can I check if it's working how I want it to? 

Sorry again for bugging you what must be quite an obvious process.

thanks
u.

**Reply by ujkanishka** [reply]
oh wait, do I just need to make a firmware update to GoPro labs?

**Reply by dnewman-gpsw** [reply]
Correct, this all about Labs Firmware

**Reply by ujkanishka** [reply]
hi, 
Thanks, the code works for me. 

Sorry but could I bother you to help me generate a code so that the camera can wake up in let's say 24 hours and 5 minutes and take a 5 minute video. Like a very similar one as the previous one, but this ensures that everyday, it wakes up at a different time. 

Sorry I did try to generate a code using the visual coding tool, but I was unable to. 

thanks
ujjwal.

**Reply by dnewman-gpsw** [reply]
You should be able to do this without the visual coding tool.

These are you actions:
Sleep(24 hours + 5 minute)
mode video 
Start record
End Record 5 minute later
Report

Each has a simple command (read https://gopro.github.io/labs/control/tech/)

!86700N  <- Sleep day and five minute - time in seconds
mV  <- mode video 
!S <- start
!300E <- End after 300 seconds.
!2R <- pause for 2 seconds then repeat

So you command is: ```!86700NmV!S!300E!2R```

**Reply by ujkanishka** [reply]
oh cool, thanks again :)
